vncviewer/parameters.cxx

```diff
@@ -181,6 +185,9 @@ static VoidParameter* parameterArray[] = {
 #endif // HAVE_GNUTLS
   &SecurityClient::secTypes,
   /* Misc. */
+  &emulateMiddleButton,
+  &emulateMiddleButton2,
+  &dotWhenNoCursor,
```

Comment:
Why did you put this in here given the scope of this commit?
Not it's in the array twice.

Reply:
Thanks for the review.

This is a mistake from the last rebase conflict resolution. I have fixed it.  Thanks for catching it.